doc/user/CONFIGURATION_FILE.md

```diff
@@ -68,9 +68,9 @@ off on the command line if needed, eg: `--point-sprites=false`.
 The configuration file or directory possible locations depends on your operating system.
 They are considered in the below order and only the first found will be used.
 
- * Linux: `${XDG_CONFIG_HOME}/.config/f3d/config(.json,.d)`, `~/.config/f3d/config(.json,d)`, `/etc/f3d/config(.json,.d)`, `/usr/share/f3d/configs/config(.json,.d)`, `[install_dir]/share/f3d/configs/config(.json,.d)`
+ * Linux: `${XDG_CONFIG_HOME}/f3d/config(.json,.d)`, `~/.config/f3d/config(.json,d)`, `/etc/f3d/config(.json,.d)`, `/usr/share/f3d/configs/config(.json,.d)`, `[install_dir]/share/f3d/configs/config(.json,.d)`
```

Comment:
I think `XDG_CONFIG_HOME` already contain the `.config`?

Reply:
correct